style/style.css

```diff
@@ -82,29 +87,201 @@ h2 {
   background-image: url(../assets/background.png);
   background-repeat: no-repeat;
   background-size: 375px 668px;
-  padding-left: 10px;
+  background-color: rgb(255, 255, 255);
   width: 375px;
   height: 718px;
+  margin: 0 auto;
+}
+
+.content ul,
+li {
+  list-style: none;
+  display: inline;
+  padding: 1px 13px;
+  margin: -3px;
 }
 
-ul,
+.contact ul,
 li {
   list-style: none;
   display: inline;
   padding: 1px 13px;
-  margin-left: -3px;
+  margin: -3px;
+}
+
+.works h2 a {
+  color: #396df2;
+  border: 2px solid #396df2;
+  border-radius: 8px;
+  padding: 10px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+.contact h2 a {
+  color: #396df2;
+  border: 2px solid #396df2;
+  border-radius: 8px;
+  padding: 10px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  margin: 16px;
+}
+
+.contact h2 a:hover {
+  background-color: #6070ff;
+  color: rgb(255, 255, 255);
+  padding: 10px;
+  border-radius: 8px;
+}
+
+.contact h2 a:active {
+  background-color: #2230d2;
+  border: 2px solid #2230d2;
+  color: rgb(255, 255, 255);
+  padding: 10px;
+  border-radius: 8px;
+}
+
+.works h2 a:hover {
+  background-color: #6070ff;
+  color: rgb(255, 255, 255);
+  padding: 10px;
+  border-radius: 8px;
 }
 
-h2 a:hover {
+.works h2 a:active {
+  background-color: #2230d2;
+  border: 2px solid #2230d2;
+  color: rgb(255, 255, 255);
+  padding: 10px;
+  border-radius: 8px;
+}
+
+#connect a:hover {
   background-color: #6070ff;
   color: rgb(255, 255, 255);
   padding: 5px;
   border-radius: 8px;
 }
 
-h2 a:active {
+#connect a:active {
   background-color: #2230d2;
   color: rgb(255, 255, 255);
   padding: 5px;
   border-radius: 8px;
 }
+
+.big {
+  height: 596px !important;
+}
+
+.works {
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  justify-content: center;
+  background-color: rgb(255, 255, 255);
+  width: 327px;
+  height: 540px;
+  border-radius: 16px;
+  margin: 44px 24px 44px 24px;
+  padding-inline-start: 1px;
+}
+
+.works img {
+  margin: 1px 15px 1px 5px;
+}
+
+.works h1 {
+  font-weight: 700;
+  color: #172b4d;
+  width: 295px;
+  height: 36px;
+  margin: 1px 15px 1px 5px;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.big h1 {
+  height: 88px;
+}
+
+.works ul li {
+  font-weight: 600;
+  color: #344563;
+  font-size: 13px;
+  display: inline;
+  height: 13px;
+  width: 279px;
+  padding: 1px 10px;
+}
+
+.works ul {
+  padding-inline-start: 1px;
+  margin: 10px 1px;
+}
+
+.qualification {
+  list-style: circle;
+  color: #7a869a;
+}
+
+.works-text {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 295px;
+  height: 332px;
+  margin: 12px 16px 16px 16px;
+}
+
+.works p {
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 24px;
+  margin: 10px 10px;
+}
+
+.languages {
+  margin: 5px 1px 45px 1px !important;
```

Comment:
- [ ] Please keep in mind that there's a rule in CSS that says that we should not overuse the `!important`, and most developers prefer not to use it at all. However, in your case, I didn't notice any difference in the project when I removed it from the code. So, it shouldn't be there.

> This [article](https://brightinventions.pl/blog/do-not-overuse-important/) about that topic is great and I recommend taking a look at it 😄

Reply:
:heavy_check_mark: Fixed.